tutorials/appengine-python-api-firebase-auth/app.yaml

```diff
@@ -0,0 +1,10 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+
+handlers:
+  - url: /.*
+    script: api.app
+
+env_variables:
+  GCLOUD_PROJECT: 'clear-beacon-145712'
```

Comment:
s/clear-beacon-145712/YOUR_PROJECT_ID/

but actually, doesn't App Engine Standard set the GCLOUD_PROJECT env var for you? I know it does in App Engine Flexible Environment